### DISCUSSION TITLE
Does it work with Arch Linux and the spotify package in the AUR?

### DISCUSSION THREAD

**Discussion by ghost** [SpotX-Official/SpotX-Bash#9] [Q&A] [answered]
Hello everyone,

I hope you all doing well. First,  I would appreciate your work and I want to extend my sincerest thanks to the developers of this project. Thank you so much, your amazing work can help many people, including those who face certain challenges in using online payment systems in some countries like mine.
In my country, people are restricted from using online payment systems such as PayPal and Visa cards and etc. due to political US laws and government filtering from US and also we are facing such huge internet filtering/blocking problems from inside of the country. In addition we also have to use spotify services itself and many other services using a proxy or vpn.
As a result, it is incredibly difficult for us to buy anything online, especially from international websites. However, I have noticed that this SpotX project could be incredibly helpful for people like me, who are looking for alternative ways to access online services like any other normal person in normal countries.

I've used Spot-X on my windows machine but I'm planning to use this project on my Arch Linux system, and I'm almost new to Linux and Bash so I was wondering if anyone could help me with a question I have regarding its compatibility with the Spotify client in Arch Linux AUR. If anyone in the community has experience using this project on Arch Linux and knows whether it is compatible with the Spotify client in Arch AUR, also I have noticed that the most recent Spotify client in the AUR has some serious issues and problems sometimes. Therefore, I was wondering if there is any better way to use the official Spotify client itself on Arch Linux.  I would greatly appreciate your help.
Thank you again to the developers for creating such a project.

**Comment by jetfir3** [comment, marked as answer]
SpotX-Bash is not distro-specific and should work with any distro that Spotify can be installed on -- its only limitation is NOT supporting Snap installs.

This project does have arch users. I cannot help with suggesting a "better" way to install Spotify as I am not an arch user. Not much talk from linux users (or any users) for this project but you may have a higher chance of a response in our telegram community (linked on readme).
